pkg/controller/perconaservermongodb/external_service.go

```diff
@@ -34,6 +35,41 @@ func (r *ReconcilePerconaServerMongoDB) ensureExternalServices(cr *api.PerconaSe
 	return services, nil
 }
 
+func (r *ReconcilePerconaServerMongoDB) exportServices(cr *api.PerconaServerMongoDB) error {
+	if !cr.Spec.MultiCluster.Enabled {
+		return nil
```

Comment:
when we disable multiCluster are services deleted?

Reply:
Deletion of outdated `ServiceExports` is implemented in bfa5bf26d2d8472a2531f519fc85ee3456042b22